### PR TITLE
Updated filter

### PR DESCRIPTION
**Pull request by tmicomyiza**
user can filter events by category.

### CODE REVIEW

**Comment by tmicomyiza** [comment]
@googlebot I consent.

**Comment by tmicomyiza** [comment]
@googlebot I consent.

**Comment by mayahp** [comment]
@googlebot I consent.

File: frontend/src/components/MapView.js [hunk 1]
```diff
@@ -192,8 +205,11 @@ class MapView extends Component {
         imageUrl = imageUrl.split(",")[0];
       }
 
+
       return(
         <EventInfoWindow
+          onOpen={this.windowHasOpened}
+          onclose={this.windowHasClosed}
```
Comment (danhoshow) [inline frontend/src/components/MapView.js:212]: I don't know if this is allowed, but the capitalization of these two lines are inconsistent

File: frontend/src/components/Profile.js [hunk 2]
```diff
@@ -49,62 +52,63 @@ class Profile extends React.Component {
...
 
-      // dynamically create a card that contain images for the event
-      card = imageUrl.map(url => <Card.Img variant="top" src={url} />);
-    }
+      // Since images are option field, evaluate whether the field isn't null
+      if (event.imagePaths != null) {
+        len = event.imagePaths.length;
+      }
+      // Retrieve image urls only if they were provided. Otherwise set url to default value
+      if(len > 0) {
```
Comment (danhoshow) [inline frontend/src/components/Profile.js:66]: this is already in an if block for `len > 0`, but should you be checking if `len > 1`?

File: frontend/src/components/Profile.js [hunk 3]
```diff
@@ -116,6 +120,22 @@ class Profile extends React.Component {
...
+    var names = []
+    var len = refAttendees.length;
+    var ref = fb.userRef.child('users');
+
+    for (var i = 0; i < len; i++) {
+      var current_attendee = ref.child(refAttendees[i]).child('name');
+
+      current_attendee.on('value', snapshot => {
+        let name = snapshot.val();
+        names.push(name);
```
Comment (danhoshow) [inline frontend/src/components/Profile.js:133]: not super familiar with firebase, is this done synchronously? Will `names` contain all the attendee's names when it's returned?

**Pull request merged**